### PR TITLE
2 list space

### LINKED ISSUE #2: #2 - List Space
As a user
so that I can make money
I want to be able to list a new space

### PR DESCRIPTION
**Pull request by rogrenke**
closes #2 
mvp style ftw!

### CODE REVIEW

File: test/spaceTest.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+var mongoose = require('mongoose'),
+    chai = require('chai'),
+    assert = chai.assert,
+    expect = chai.expect;
+
+mongoose.Promise = global.Promise;
+mongoose.connect('mongodb://localhost/makersbnb_test');
```
Comment (marioribeiro) [inline test/spaceTest.js:7]: Removed empty lines _(testing purposes)_

**Review by rogrenke — COMMENTED** [review]
Really good! Thank you :)

**Review by marioribeiro — APPROVED** [review]
Approved

**Comment by marioribeiro** [comment]
:shipit:

**Pull request merged**